install.ps1

```diff
@@ -0,0 +1,108 @@
+# Installs KoreBuild into a repo, overwriting all previous build scripts
```

Comment:
We're probably going to bulk update repos as a one time (basically how @victorhurdugaci updated to the current KoreBuild). Wouldn't we just do this offline (without having to commit this)?